addons/binding/org.openhab.binding.vallox/README.md

```diff
@@ -0,0 +1,143 @@
+# Vallox Binding
+
+Interface to a vallox central venting unit.
+
+Tested with 
+* Vallox KWL 90 SE and 
+* USR-TCP232-410 RS485-to-ethernet interface
+
+## Prerequisite
+
+This Openhab2 bundle so far only works with a RS485-to-ethernet gateway.
+It does not support a local serial interface of the device executing OH2.
+
+I do not have any experience with direct serial RS485 communication under Java and
+do not have any device for testing it. So if you can contribute any code for direct
+RS485 serial write and read, I might easily merge that.
+
+## Configuration
+
+Configure the serial interface such that it acts as TCP server on some port of your choice. 
+Configure the serial side as 
+* baudrate: 9600
+* data size: 8 bit
+* parity: none
+* stop bits: 1 
+
+## Install
+
+1. open paper ui in browser
+2. Goto Configuration -> System and disable Item Linking Simple mode (so no items will be created automatically)
+3. check that Configuration -> Bindings lists the Vallox binding
+4. go to Configuration -> Things
+5. add Vallox KWL Serial Binding Thing (exactly one)
+6. add textual configuration (see below) or enter configuration parameters (host and port of serial gateway)
+7. The thing should show status `ONLINE`; if not, go into the details of the thing. The status there should include some error message.
+8. make sure you create items for all the many channels of the binding
+  * see items example below
+  * after adding explicit items to your items file, you need to link them to the vallox thing channels (see below for textual example)
+  * managing the items manually allows you to correctly reference the items in rules and to group them in groups
+9. make the items available in some sitemap and you should be able to control the vallox via the OH2 UIs
+  * see example below
+
+## Features
+
+The binding currently supports the following Features
+* 14 major properties, some read-only, some writeable
+* 35 advanced properties, so far read-only most of them
+
+Note that there might be more properties that can be read or written which might be not implemented yet. Contributions welcome.
+Also not all channels are perfectly documented. See [thing-types.xml](ESH-INF/thing/thing-types.xml) for details.
+
+* the binding will automatically detect loss of connection (at latest by a heartbeat poll to the vallox every minute)
+* the binding will try to reconnect regularly until connection is re-established
+
+## Examples
+
+### vallox.things
+
+It's easiest to define the thing textually and map the items textually to the thing's channels.
+Parameters are
```

Comment:
Those should be mentioned in a "Thing configuration" section above, not only here under "Examples".
Check the official template for the binding README.